classes/Task/Update/UpdateDatabase.php

```diff
@@ -170,7 +170,7 @@ protected function checkVersionIsNewer(): void
         $versionCompare = version_compare($destinationVersion, $currentVersion);
 
         if ($versionCompare === -1) {
-            throw new UpgradeException($this->container->getTranslator()->trans('[ERROR] Version to install is too old.') . ' ' . $this->container->getTranslator()->trans('Current version: %oldversion%. Version to install: %newversion%.', ['%oldversion%' => $currentVersion, '%newversion%' => $destinationVersion]));
+            throw new UpgradeException($this->container->getTranslator()->trans('Version to install is too old.') . ' ' . $this->container->getTranslator()->trans('Current version: %oldversion%. Version to install: %newversion%.', ['%oldversion%' => $currentVersion, '%newversion%' => $destinationVersion]));
```

Comment:
Can we take the opportunity to merge these texts?

```suggestion
            throw new UpgradeException($this->container->getTranslator()->trans('Version to install is too old. Current version: %oldversion%. Version to install: %newversion%.', ['%oldversion%' => $currentVersion, '%newversion%' => $destinationVersion]));
```